src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
it's a good practice to use Object Destructuring
```suggestion
export const Person = ({ person }) => (
  const { name, age, isMarried, partnerName, sex } = person;
  
  return (
    <section className="Person">
      {{your__code}}
    </section>
  )
)
```